plugins/modules/discovery_intent.py

```diff
@@ -721,7 +736,7 @@ def get_discovery_device_info(self, discovery_id=None, task_id=None):
             )
             devices = response.response
 
-            self.log(devices)
+            self.log("Devices Information retrieved from the get discovered network devices by discovery id API is {0}".format(str(devices)), "DEBUG")
```

Comment:
self.log("Retrieved device details using the API 'get_discovered_network_devices_by_discovery_id': {0}".format(str(devices)), "DEBUG")

Reply:
Resolved